tools/cli/go-whisk-cli/commands/namespace.go

```diff
@@ -119,8 +119,10 @@ var listCmd = &cobra.Command{
 }
 
 func init() {
+    listCmd.Flags().BoolVarP(&flags.namespace.date, "sort", "t", false, wski18n.T("sort by date"))
```

Comment:
Yes, just to address issue #1569, ok, I'll try to change a better one for your review.